test/fixtures/es-module-loaders/mock.mjs

```diff
@@ -0,0 +1,71 @@
+import * as fixtures from '../../common/fixtures.mjs';
+import { register } from 'node:module';
+import { MessageChannel } from 'node:worker_threads';
+
+
+const { port1, port2 } = new MessageChannel();
+
+register(fixtures.fileURL('es-module-loaders/mock-loader.mjs'), {
```

Comment:
nit: IMO most folks will use a relative URL here
```suggestion
register('./mock-loader.mjs', import.meta.url, {
```

Reply:
It has to be absolute or we pass parent URL.